client/components/widget/data_viz/gviz/gviz-directive.html

```diff
@@ -1,14 +1,20 @@
 <div>
-  <div class="pk-chart"
-       ng-show="isChartDisplayed()">
-  </div>
-  <div layout="row" layout-sm="column" layout-align="space-around"
-       class="pk-chart-loading md-primary md-hue-3"
-       ng-show="isLoadingDisplayed()">
-    <md-progress-circular md-mode="indeterminate"></md-progress-circular>
-  </div>
-  <div class="pk-chart-error"
-       ng-show="isErrorDisplayed()">
-    {{ widgetConfig.state().chart.error }}
+  <div class="pk-conditional" ng-class="getWidgetStatusClass()">
+    <div class="pk-if pk-cond-nodata pk-chart-nodata">
+      No data available.
```

Comment:
This div needs styled to be similar to the chart error styling (centered, italic, greyed, etc.).